examples/mnit_ballz/src/assets.nit

```diff
@@ -0,0 +1,49 @@
+# this file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# All the assets needed by the appplication, regrouped in one class
+module assets
+
+import app::audio
+import mnit::assets
+
+
+# Conttains all the assets
+class Assets
+
+	# Sound for the wall destruction
+	var wall_destruction = new Sound("walld")
+
+	# Sound when the ball bounce
+	var bounce = new Sound("bounce")
```

Comment:
There is a little problem here : 
It doesn't work because on linux because of the extension, that's for sure.

The problem is, the android Sound API tries to load a sound from assets first, and it needs the file extension. It then tries to load a sound from the res/raw folder, and it doesn't work with the extension on the file. I thought about using `strip_extension` when it comes to load the sound from the resources, but then I walked into another problem : 
Linux assets are located in the `assets` folder, nothing abnormal here. Android assets are in the assets folder or the resources arborescence. So when you want to make an application that runson both android and Linux, but you want to use the resources on android, you'll always be loading the sounds from the assets folder since the Android Sound API tries it first.

Got 2 solutions for that : 
* Reverse the loading order of the sound in the Android API
* Just use the `strip_extension` trick, but then you'll never load your sounds from somewhere else than the assets folder when you're making an application that runs on both linux and android

Well after writing all of that, I think i'll just use the first solution \o/